moon/map.conf

```diff
@@ -7,10 +7,10 @@ rotation = z
 
 team.1 = red
 team.1.color = red
-team.1.pos = (-96,-25,97)
```

Comment:
You probably exported map wrongly when you finished editing
![Zaslonska slika 2020-02-03 16-50-22](https://user-images.githubusercontent.com/47271658/73668001-71bd4900-46a5-11ea-91e5-39a19647917a.png)
 
You should place //1 exactly in corner of the map.

Reply:
Or did you use /gui command to export it?

Reply:
Base coordinates shouldn't change.

Reply:
Buildings look great. Do you mind if I mention (in author line) you and gimp made buildings?

Reply:
I had some time today and fixed this.